src/js/controllers/walletHome.js

```diff
@@ -28,6 +28,27 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
   this.addr = {};
   this.lockedCurrentFeePerKb = null;
 
+  self.hideBalance = function() {
+    var fc = profileService.focusedClient;
+    storageService.getHideBalanceFlag(fc.credentials.walletId, function(err, shouldHideBalance) {
+      if (err) self.shouldHideBalance = false;
+      else self.shouldHideBalance = shouldHideBalance ? true : false;
```

Comment:
This should be `self.shouldHideBalance = (shouldHideBalance == 'true') ? true : false;` because I receive a string value from local storage.